test/controllers/detailedvaluationrequest/DvrControllerSpec.scala

```diff
@@ -64,6 +70,14 @@ class DvrControllerSpec extends VoaPropertyLinkingSpec {
     val assessment = arbitrary[Assessment].sample.get
     val link: PropertyLink = arbitrary[PropertyLink]
 
+    lazy val successfulDvfDocuments: Future[Some[DvrDocumentFiles]] = Future.successful(Some({
```

Comment:
`successfulDvfDocuments` typo - just to show I'm paying attention!

Reply:
🦅 👁️